pkg/api/image/formatter.go

```diff
@@ -119,7 +119,8 @@ func (h UploadActionHandler) uploadImage(rw http.ResponseWriter, req *http.Reque
 		return fmt.Errorf("failed to read response body: %w", err)
 	}
 	if uploadResp.StatusCode >= http.StatusBadRequest {
-		return fmt.Errorf("upload failed: %s", string(body))
+		err = fmt.Errorf("upload failed: %s", string(body))
+		return err
```

Comment:
why not return directly instead.

Reply:
it will be handled in https://github.com/harvester/harvester/pull/1507/files#diff-0d92d61b314f651cb0a00c56ea66f8b5012681711fff3a8618439da5b747b50aR83-R89

Reply:
can we add a comment to this one to avoid maloperation in the future, thanks?